packages/client/verifiable-credentials/src/presentation/getNfts.ts

```diff
@@ -41,6 +41,29 @@ export async function getWalletNfts(chain: string, wallet: string) {
     return allData;
 }
 
+export async function getWalletVCNfts(chain: string, wallet: string): Promise<CrossmintWalletNft[]> {
+    const baseUrl = crossmintAPI.getBaseUrl();
+    const headers = crossmintAPI.getHeaders();
+
+    const url = `${baseUrl}/api/v1-alpha1/wallets/${chain}:${wallet}/nfts/credentials`;
```

Comment:
Depends on the discussion on https://github.com/Paella-Labs/crossbit-main/pull/14192